requirements/tox-static-code-analysis.in

```diff
@@ -0,0 +1,7 @@
+Django>=3.2.9
+psycopg2
+exam
+pytest
+pytest-cov
+pytest-django
+prospector[with_everything]
```

Comment:
I could not add comments on tox.in (below file) so I'm adding here
This file is the previous file used for dependency management
It is now being used for input file for tox-requirements.txt